vesuvius-c.h

```diff
@@ -4546,53 +4763,5 @@ slice *vs_tiff_to_slice(const char *tiffpath, int index) {
 }
 
 
-
-int vs_slice_fill(slice *slice, volume *vol, int start[static 2], int axis) {
```

Comment:
these methods were left over from some old stuff and are superseded by vs_vol_get_chunk

Reply:
addressed